c++/src/capnp/layout.c++

```diff
@@ -30,26 +30,47 @@
 #include "capability.h"
 #endif  // !CAPNP_LITE
 
+#ifdef __has_feature
+#define IS_BUILDING_WITH_TSAN() __has_feature(thread_sanitizer)
+#else
+#if defined(__SANITIZE_THREAD__) && __SANITIZE_THREAD__
+#define IS_BUILDING_WITH_TSAN() 1
+#else
+#define IS_BUILDING_WITH_TSAN() 0
+#endif
+#endif
+
+
 namespace capnp {
 namespace _ {  // private
 
 #if !CAPNP_LITE
-static BrokenCapFactory* brokenCapFactory = nullptr;
+static BrokenCapFactory* gBrokenCapFactory = nullptr;
```

Comment:
This `g` prefix isn't idiomatic KJ, but I'd accept `globalBrokenCapFactory`.

Reply:
Sorry about that. Switching between different codebases.  Definitely prefer KJ's style.